components/SuggestedColor/index.js

```diff
@@ -0,0 +1,5 @@
+import { StyledDiv } from "./styles";
+
+export default function SuggestedColor({ text, color }) {
+  return <StyledDiv color={color}>{text}</StyledDiv>;
```

Comment:
you could change this to {children}.
also this is only used to have a styled Div at this point? 
so you don't really need a component here, but you could style it directly on your suggestions-page

Reply:
I implemented the `children`. I used the component and the styles.js for future US and to tidy up a little